src/main/scala/com/timeout/healthchecks/HealthChecks.scala

```diff
@@ -0,0 +1,20 @@
+package com.timeout.healthchecks
+
+import java.net.{Socket, URL}
+
+import cats.syntax.all._
+
+import scala.util.Try
+
+object HealthChecks {
+
+  def ping(host: String, port: Int) = healthCheck(s"Ping $host:$port", () => {
```

Comment:
Could we also get into the habit of having signature for public functions? It makes it easier to review and again decreases compilation time (I think)